components/AlertCard/AlertCard.tsx

```diff
@@ -35,3 +35,42 @@ export const AlertCard = (props: {
 
   return <MapleCard headerElement={header} body={body} />
 }
+
+// newsfeed bill card
+export const AlertCardV2 = (props: {
```

Comment:
nit: Could we give this a more descriptive name than `AlertCardV2`? `NewsfeedBillCard` would work for me for now - we can always generalize the card and name if someone ever needs to use the design in another context.